src/liquid/components/ld-cookie-consent/ld-cookie-consent.tsx

```diff
@@ -0,0 +1,789 @@
+      this.showDisclaimer(this.config.showOnLoadDelay)
+    }
+  }
+
+  render() {
+    const { config } = this
+    return (
+      <Host class="ld-cookie-consent">
+        {/* Disclaimer */}
+        <div
+          aria-labelledby="ld-cookie-consent-disclaimer-header"
+          class={getClassNames([
+            'ld-cookie-consent__disclaimer',
+            `ld-cookie-consent__disclaimer--${config.disclaimerAlignement}`,
+            config.rejectable && 'ld-cookie-consent__disclaimer--rejectable',
+            this.isDisclaimerVisible &&
+              'ld-cookie-consent__disclaimer--visible',
+          ])}
+          onTransitionEnd={function () {
+            this.focus()
+          }}
+          part="disclaimer"
+          role="dialog"
+          tabIndex={this.isDisclaimerVisible ? -1 : undefined}
+        >
+          <div
+            class="ld-cookie-consent__disclaimer-header"
+            part="disclaimer-header"
+          >
+            <div
+              class="ld-cookie-consent__disclaimer-logo-container"
+              part="disclaimer-logo-container"
+            >
+              <slot name="disclaimer-logo">{this.getInitialM()}</slot>
+            </div>
+
+            <slot name="disclaimer-title">
+              <ld-typo
+                class="ld-cookie-consent__disclaimer-title"
+                part="disclaimer-title"
+                tag="p"
+                variant="h5"
+              >
+                Cookie Disclaimer
+              </ld-typo>
+            </slot>
+
+            {config.dismissable && (
+              <button
+                aria-label={this.config.buttonDismiss}
+                class="ld-cookie-consent__disclaimer-x"
+                onClick={this.hideDisclaimerAndPreferences.bind(this)}
+                part="disclaimer-button-dismiss"
+              ></button>
+            )}
+          </div>
+
+          <div
+            class="ld-cookie-consent__disclaimer-scroll-container"
+            id="ld-cookie-consent-disclaimer-header"
+            part="disclaimer-scroll-container"
+          >
+            <div
+              class="ld-cookie-consent__disclaimer-description-container"
+              part="disclaimer-description-container"
+            >
+              <slot name="disclaimer-description">
+                {config.mode === 'notice-only' && (
+                  <ld-typo
+                    class="ld-cookie-consent__disclaimer-description"
+                    part="disclaimer-description"
+                  >
+                    This website uses cookies so that you have the best user
+                    experience. By continuing your browsing on this website, you
+                    accept the conditions described in our{' '}
+                    <ld-link href={config.privacyStatementURL} target="_blank">
+                      Cookie Policy / Privacy Statement
+                    </ld-link>
+                    . Cookies can be managed using your browser preferences.
```

Comment:
That's the default slot content, so it can be overwritten already. I decided to implement it this way, so that inlined links (mainly the one to the privacy policy) can be placed freely in each language.